documentation/modules/oauth/con-oauth-authentication-flow.adoc

```diff
@@ -35,7 +45,18 @@ The client must enable SASL_PLAIN and set the `username` and `password`.
 If the client sets `username` to `access-token`, the authorization server will interpret the password as the access token. 
 Otherwise, the authorization server will interpret the user as the `clientId` and the password as the client secret.
 
-Kafka brokers and clients both need to be configured to use OAuth 2.0. SASL_PLAIN must be enabled in the listener configuration for the Kafka broker.
+Kafka brokers and clients both need to be configured to use OAuth 2.0. SASL_PLAIN must be enabled in the listener OAuth 2.0 authentication configuration for the Kafka broker.
+
+[source,yaml,subs="+quotes,attributes+"]
+----
+  # ...
+  authentication:
+    type: oauth
+    # ...
+    enableOauthBearer: false
+    enablePlain: true
```

Comment:
Will it be clear tot he users that they can also have both set to true?

Reply:
Probably not...

I can add something like this:

SASL_OAUTHBEARER and SASL_PLAIN can be enabled at the same time to support both types of clients.

Reply:
Added:

> OAUTHBEARER and PLAIN can be enabled at the same time, on the same `oauth` listener, if both types of OAuth clients need to authenticate.